spec/factories/topics.rb

```diff
@@ -2,6 +2,7 @@
 FactoryGirl.define do
   factory :topic do
     user
+    association :updated_by, factory: :user
```

Comment:
these updated_by fields should be checked somewhere, right? i don't think that's currently happening